### ISSUE TITLE
Scaling breaks when dragging window between different DPI monitors

### ISSUE THREAD

**Issue by peterhellberg** [faiface/gui#5]
Moving the window from a lower DPI monitor to a higher DPI monitor results in broken rendering:
<img width="912" alt="gui-example-paint-broken-scaling" src="https://user-images.githubusercontent.com/565124/57376192-b613cb00-719f-11e9-814f-822464300219.png">

(The cursor position is still reported correctly)

**Comment by faiface** [comment]
Yeah, thanks for reporting! I'll take a look at how I can fix this. If you have a solution already, please tell me! :D
